### PR TITLE
Automated Test - COMMERCE-6160 | master

### PR DESCRIPTION
**Pull request by brunadmelo**

### CODE REVIEW

**Comment by brunadmelo** [comment]
https://issues.liferay.com/browse/COMMERCE-6160; cc:@LorenzoLaPescara

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -55,12 +55,51 @@ definition {
...
+		ApplicationsMenu.gotoSite(site = "Minium");
+
+		Click(
+			key_productName = "U-Joint",
+			locator1 = "CommerceAccelerators#PRODUCT_CARD_QUANTITY_SELECTOR_SELECT_FIELD");
+
+		AssertElementPresent(
+			key_dropdownOption = "U-Joint",
+			locator1 = "AppBuilder#PRODUCT_MENU_SITE_DROPDOWN_OPTION",
+			value1 = "1 4 5 7 11");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:102]: The locator points at the Product Card name and this line it's not actually checking that the values on the quantity selector are only the allowed ones.
Please try to find a different way to check product card quantity selector option values.

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 2]
```diff
@@ -55,12 +55,52 @@ definition {
...
+
+		CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+		CommerceNavigator.gotoEntry(entryName = "U-Joint");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Configuration");
+
+		AssertElementPresent(
+			key_labelName = "Allowed Order Quantities",
+			locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT");
```
[the remark this answers is not in the document]
Comment (brunadmelo) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: Done.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 3]
```diff
@@ -55,12 +55,52 @@ definition {
...
+
+		Type(
+			key_labelName = "Allowed Order Quantities",
+			locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT",
```
[the remark this answers is not in the document]
Comment (brunadmelo) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: Done.

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TextInput.path [hunk 4]
```diff
@@ -688,6 +688,11 @@
 	<td>//input[contains(@id,'webId') and contains(@class,'input-text')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SPECIFIC_LABEL_TEXT_INPUT</td>
+	<td>//label[contains(text(),'${key_labelName}')]//following::input[contains(@type,'text')]</td>
+	<td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (brunadmelo) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TextInput.path]: Removed.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 5]
```diff
@@ -55,12 +55,52 @@ definition {
...
+			portlet = "Products");
+
+		CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+		CommerceNavigator.gotoEntry(entryName = "U-Joint");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Configuration");
+
+		AssertElementPresent(
+			key_labelName = "Allowed Order Quantities",
```
[the remark this answers is not in the document]
Comment (brunadmelo) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: Done.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 6]
```diff
@@ -55,12 +55,52 @@ definition {
...
+			locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT");
+
+		Type(
+			key_labelName = "Allowed Order Quantities",
```
[the remark this answers is not in the document]
Comment (brunadmelo) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: Done.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase (hunk 1 above)
Comment (brunadmelo) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: Done.

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:forward

**Pull request closed without merging**